src/main/java/com/thalesgroup/hudson/plugins/sourcemonitor/SourceMonitorResult.java

```diff
@@ -49,7 +49,19 @@ public SourceMonitorReport getReport(){
         return owner;
     }
 
-
+    public String getDetailedMetrics(){
+        StringBuilder builder = new StringBuilder();
+        for (int i=0; i<report.getDetailedMetrics().size();i++){
+            builder.append("<tr><td>");
```

Comment:
What happens when you load an report where detailed metrics has not been set? Do you get a blank table or a null exception?

Reply:
Thanks. Added if in jelly file that works with old/new.